actors/builtin/miner/miner_state.go

```diff
@@ -309,46 +301,185 @@ func (st *State) ForEachSector(store adt.Store, f func(*SectorOnChainInfo)) erro
 
 // Adds some sector numbers to the new sectors bitfield.
 func (st *State) AddNewSectors(newSectors ...abi.SectorNumber) error {
-	panic("Implement me") // TODO
+	for _, sector := range newSectors {
+		st.NewSectors.Set(uint64(sector))
+	}
+
+	return nil
 }
 
 // Clears the new sectors bitfield.
 func (st *State) ClearNewSectors() error {
-	panic("implement me") // TODO
+	st.NewSectors = abi.NewBitField()
+	return nil
 }
 
 // Adds some sector numbers to the set expiring at an epoch.
 func (st *State) AddSectorExpirations(store adt.Store, expiry abi.ChainEpoch, newSectors ...abi.SectorNumber) error {
-	panic("implement me") // TODO
+	arr, err := adt.AsArray(store, st.SectorExpirations)
+	if err != nil {
+		return err
+	}
+
+	bf := abi.NewBitField()
+	_, err = arr.Get(uint64(expiry), &bf)
+	if err != nil {
+		return err
+	}
+
+	for _, sector := range newSectors {
+		bf.Set(uint64(sector))
+	}
```

Comment:
Ditto

Reply:
Here this doesn't matter as we're immedietally putting the bitfield into an amt